client/me/security-account-recovery/index.jsx

```diff
@@ -61,48 +62,55 @@ const SecurityAccountRecovery = ( props ) => (
 
 		<DocumentHead title={ props.translate( 'Account Recovery' ) } />
 
-		<CompactCard>
+		<Card>
 			<p className="security-account-recovery__text">
 				{ props.translate(
 					'Keep your account safe by adding a backup email address and phone number. ' +
 						'If you ever have problems accessing your account, WordPress.com will use what ' +
 						'you enter here to verify your identity.'
 				) }
 			</p>
-		</CompactCard>
+		</Card>
 
-		<CompactCard>
-			<RecoveryEmail
-				primaryEmail={ props.primaryEmail }
-				email={ props.accountRecoveryEmail }
-				updateEmail={ props.updateAccountRecoveryEmail }
-				deleteEmail={ props.deleteAccountRecoveryEmail }
-				isLoading={ props.accountRecoveryEmailActionInProgress }
-			/>
-			{ props.shouldPromptEmailValidationNotice && ! props.hasSentEmailValidation && (
-				<RecoveryEmailValidationNotice
-					onResend={ props.resendAccountRecoveryEmailValidation }
-					hasSent={ props.hasSentEmailValidation }
+		<div className="security-account-recovery__recovery-email">
+			<SectionHeader label={ props.translate( 'Recovery email address' ) } />
+			<Card>
+				{ props.shouldPromptEmailValidationNotice && ! props.hasSentEmailValidation && (
+					<RecoveryEmailValidationNotice
+						onResend={ props.resendAccountRecoveryEmailValidation }
+						hasSent={ props.hasSentEmailValidation }
+					/>
+				) }
+				<RecoveryEmail
+					primaryEmail={ props.primaryEmail }
+					email={ props.accountRecoveryEmail }
+					updateEmail={ props.updateAccountRecoveryEmail }
+					deleteEmail={ props.deleteAccountRecoveryEmail }
+					isLoading={ props.accountRecoveryEmailActionInProgress }
 				/>
-			) }
-		</CompactCard>
+			</Card>
+		</div>
 
-		<CompactCard>
-			<RecoveryPhone
-				phone={ props.accountRecoveryPhone }
-				updatePhone={ props.updateAccountRecoveryPhone }
-				deletePhone={ props.deleteAccountRecoveryPhone }
-				isLoading={ props.accountRecoveryPhoneActionInProgress }
-			/>
-			{ props.shouldPromptPhoneValidationNotice && (
-				<RecoveryPhoneValidationNotice
-					onResend={ props.resendAccountRecoveryPhoneValidation }
-					onValidate={ props.validateAccountRecoveryPhone }
-					hasSent={ props.hasSentPhoneValidation }
-					isValidating={ props.validatingAccountRecoveryPhone }
+		<div className="security-account-recovery__recovery-sms-number">
+			<SectionHeader label={ props.translate( 'Recovery phone number' ) } />
```

Comment:
Should we keep `Recovery SMS number`?

Reply:
No strong opinions, but we're using both terms "phone" and "SMS" so I changed it to phone number so that it matches the "phone number" field:

![image](https://github.com/user-attachments/assets/caa9e644-9c3a-4e6f-8e4c-1e883fb8c3d0)

Reply:
Ok, but then we’ll need to wait for the new translation 🙂

Reply:
Ooops you're right, I'll change it back to Recovery SMS number to be consistent with the previous screen:

![Screenshot 2025-05-12 at 11 32 35 AM](https://github.com/user-attachments/assets/45a78d36-880d-476e-8269-aa8d82fc0a89)